packages/sdk/test/e2e/node/apiClient.spec.ts

```diff
@@ -43,4 +43,25 @@ describe("ApiClient Tests - Node", () => {
     assert.equal(res.data.url, "/foo");
     assert.equal(res.data.requestHeader!["authorization"], "Bearer test-bearer-token");
   });
+
+  it("can connect to existing API using bearer basic auth provider", async function () {
+    // Arrange
+    const username = "test-username";
+    const password = "test-password";
+    const basicAuthProvider = new BasicAuthProvider(username, password);
+    const apiClient = createApiClient(apiBaseUrl, basicAuthProvider);
+
+    // Act
+    const res = await apiClient.get("/foo");
+
+    // Assert
+    assert.equal(res.data.url, "/foo");
+    const token = res.data.requestHeader!["authorization"].split(/\s+/).pop() || "";
```

Comment:
let's also check the value starts with `Basic`

Reply:
Fixed.